{
  "latest_version": [
    "1.1.2"
  ],
  "meta": {
    "description": "199fix exception logger for Django",
    "homepage": "https://github.com/199fix/199fix",
    "license": "MIT"
  },
  "versions": {
    "1.1.0": {
      "sha256": "c40ca01e2083c30b541975312945b4f723ea295d989ef28d7e37342cb4826187",
      "url": "https://files.pythonhosted.org/packages/f9/d4/a11111f0084352c25a2e3fe47875a91f2b71ab928d855f50402823c087c2/199Fix-1.1.0.tar.gz"
    },
    "1.1.1": {
      "sha256": "517a2fe01fa58b2088b6990d92186c8bb5214622b8ae00548e5e814508911f56",
      "url": "https://files.pythonhosted.org/packages/5a/a8/f6f2a43f9d6a08f8ef768d4ba068ee7eec9fc13863a9bab4118d74660270/199Fix-1.1.1.tar.gz"
    },
    "1.1.2": {
      "sha256": "05f8b3aa807c6d1f27d6fbba7af9fc2f9cd8cbafbb13b5440f29ec9fb85ba548",
      "url": "https://files.pythonhosted.org/packages/b0/08/009a1724b20c12ca35f3b8022db7df53ec6fb94826eb036d8b501f94a2e1/199Fix-1.1.2.tar.gz"
    }
  }
}